app/javascript/store/modules/froggo_team/actions.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+import {
+    CREATE_NEW_FROGGO_TEAM,
+} from '../../action-types';
+
+export default {
+  [CREATE_NEW_FROGGO_TEAM](_,{ name, organization_id, users }) {
+    axios.post(
+      `/api/organizations/${organization_id}/froggo_teams`,
+      { name },
+      {headers: {"Content-Type": "application/json"} },
+    )
+    .then(async (response) => {
+        const { data: { id } } = response;
+        return Promise.all(users.map(user => axios.post(`/api/froggo_teams/${id}/add_member`, { github_login: user.login })))
```

Comment:
Line 16 exceeds the maximum line length of 120                                                                 max-len<br>Expected newline before return statement                                                                       newline-before-return<br>Identifier 'github_login' is not in camel case                                                                 camelcase<br>Missing semicolon                                                                                              semi